Prevent button from refocusing on menu close

aswallace opened the pull request:
### Description
The tooltip was reappearing on buttons with menus after the menu was closed because the dom was refocusing on the original button. This prevents refocusing on menu close via [a FluentUI Callout property](https://developer.microsoft.com/en-us/fluentui#/controls/web/callout)

### Testing
Tested manually 

closes #301

BrianWhitneyAI reviewed (approved):
This also happens for the help menu but looks good for the data source button

aswallace commented:
> This also happens for the help menu 

Fixed in [d7e9e99](https://github.com/AllenInstitute/biofile-finder/pull/336/commits/d7e9e998ad4c6dae20b36d91338e4e0082a012fe): Moved the attribute to `useButtonMenu` so that it applies to all button menus
